package.json

```diff
@@ -53,7 +53,7 @@
     "test-webview": "jest",
     "posttest": "mongodb-runner stop --port=27018",
     "vscode:prepublish": "npm run clean && npm run compile:keyfile && npm run compile:resources && webpack --mode production",
-    "check": "mongodb-js-precommit './src/**/*{.ts}'",
+    "check": "npm run lint && mongodb-js-precommit './src/**/*.{js,ts,tsx}'",
```

Comment:
nice, so this works with ts too, also the depcheck?

Reply:
The `mongodb-js-precommit './src/**/*.{js,ts,tsx}'` part is specifying where it's going to lint (which is failing rn - and I don't think it lints typescript anyway). I'll open a pr adding a parameter to optionally skip linting in that package in a sec.

Reply:
The depcheck places are specified in package.json`dependency-check.entries`

Reply:
https://github.com/mongodb-js/precommit/pull/75 <- Adds an argument to `mongodb-js-precommit` to skip the lint stage.

Reply:
Oh now i get what is that for! 😄 

Why are we even using `mongodb-js-precommit`? it doesn't work with ts for eslint and depcheck anyway. Isn't it massively simpler to just add `eslint` and `depcheck` right in this project?

Reply:
yup it's easier lol. I'll update this pr to remove `mongodb-js-precommit`.